platform/base.go

```diff
@@ -123,3 +132,14 @@ func (bc *baseCluster) GetDiscoveryURL(size int) (string, error) {
 	}
 	return string(body), nil
 }
+
+// newName returns a random name prefixed by base.
+func newName(base string) (string, error) {
+	randBytes := make([]byte, 16) //128 bits of entropy
```

Comment:
I just moved that code from the GCE platform. I'm not partial to UUIDs and kinda liked the idea of using a timestamp. Do you have a preference?